jest.config.js

```diff
@@ -1,14 +1,15 @@
 module.exports = {
   rootDir: 'spec',
   transformIgnorePatterns: [
-    'node_modules/(?!(@18f/identity-address-search)/)',
+    'node_modules/(?!(@18f/identity-address-search|@18f/identity-i18n)/)',
```

Comment:
This is probably fine since we already compiled the files. This will skip having Jest attempt to compile the files.